src/login/components/App/DashboardApp/Settings/Groups/Groups.js

```diff
@@ -33,7 +34,25 @@ class Groups extends Component {
   constructor(props) {
     super(props);
     // will start as true and permanently be set to false when wizard is completed
-    this.state = { wizardCreateGroup: true};
+    this.state = { firstGroup: true };
+    this.renderCreateButton = this.renderCreateButton.bind(this);
+  }
+
+  renderCreateButton() {
```

Comment:
could we change to arrow function ` renderCreateButton()`  -> `renderCreateButton=()=>`

Reply:
👍